task_jsons/spot_bike_env/task1.description

```diff
@@ -0,0 +1,2 @@
+The hammer and hex key are in the low rack, and the screwdriver and brush are on the table.
```

Comment:
this is just for our convenience. unfortunately needs to be a separate file because JSON files can't do comments